src/operators.jl

```diff
@@ -4,6 +4,12 @@ importall Base.Operators
 import Base: promote_op, abs, abs2, sqrt, cbrt, scalarmin, scalarmax, isless
 using Compat: @functorize
 
+if isdefined(Base, :fieldname) && Base.fieldname(Nullable, 1) == :hasvalue # Julia 0.6
+    Nullable(R, x, hasvalue::Bool) = Nullable{R}(x, hasvalue)
```

Comment:
Do we need these? I'm loathe to define new methods for `Nullable`, including constructors, in this package.